framework/ui/uimodule.cpp

```diff
@@ -59,6 +60,8 @@ void UiModule::registerUiTypes()
     qmlRegisterUncreatableType<QmlTheme>("MuseScore.Ui", 1, 0, "QmlTheme", "Cannot create a QmlTheme");
     qmlRegisterUncreatableType<QmlToolTip>("MuseScore.Ui", 1, 0, "QmlToolTip", "Cannot create a QmlToolTip");
     qmlRegisterUncreatableType<IconCode>("MuseScore.Ui", 1, 0, "IconCode", "Cannot create an IconCode");
+    qmlRegisterUncreatableType<MusicalSymbolIconCode>("MuseScore.Ui", 1, 0, "MusicalSymbolIconCode",
```

Comment:
I think it's redundant to mentions "icon" if we are using the "symbol" term here
So I'd suggest to rename it for "MusicSymbolCodes"

But that is just a one-man-opinion

Reply:
agree with you